lib/fileOperations.js

```diff
@@ -196,11 +198,55 @@ function writeFile(path, data, options, callback) {
   }
 }
 
+function mkdirp(dirpath, mode, callback) {
+  if (typeof mode === 'function') {
+    callback = mode;
+    mode = undefined;
+  }
+
+  var m = mode || DEFAULT_DIR_MODE;
+  var cb = callback || function() {};
+  dirpath = path.resolve(dirpath);
+
+  fs.mkdir(dirpath, m, function(er) {
```

Comment:
Also, I was thinking it was still using the `mkdirp` module, totally forgot that the function was named mkdirp (ignore my comments about that).